### DISCUSSION TITLE
MedicationStatement

### DISCUSSION THREAD

**Discussion by dt-r** [hl7au/au-fhir-core#10] [AU Core R1 Resource Priority List - CLOSED]
Upvote for inclusion of a Resource Profile of [MedicationStatement](http://hl7.org/fhir/R4/medicationstatement.html) Resource in AU Core Release 1.

Comment if you think we need a different profile to AU Core MedicationStatement, or if there is a need for an additional MedicationStatement profile in AU Core.

AU Core MedicationStatement profile would set minimum expectations for a MedicationStatement resource to record, search, and fetch summary statements of medication use associated with a patient. 

AU Core MedicationStatement should be based on the [AU Base MedicationStatement](http://build.fhir.org/ig/hl7au/au-fhir-base/StructureDefinition-au-medicationstatement.html) profile and identify the additional mandatory core elements, extensions, vocabularies and value sets that SHALL be present in the MedicationStatement resource when conforming to this profile. It provides the floor for standards development for specific uses cases in an Australian context.

**Comment by heathfrankel** [comment]
MedicationStatement is obsolete in R5

**Reply by davidmckillop-ADHA** [reply]
[MedicationStatement](http://hl7.org/fhir/R5/medicationstatement.html) is an R5 resource, but what confuses me is the new comment of :
"_The MedicationStatement resource was previously called MedicationStatement._" in the R5 resource which is redundant text.

I understand there was some discussion on "MedicationUsage" replacing MedicationStatement, but that didn't get up.
I've added a Jira ticket [FHIR-42701](https://jira.hl7.org/browse/FHIR-42701) to ask the question on why that sentence was added to the R5 MedicationStatement resource.

**Reply by davidmckillop-ADHA** [reply]
FYI - [FHIR-42701](https://jira.hl7.org/browse/FHIR-42701) has been triaged and the comment:
"The MedicationStatement resource was previously called MedicationStatement." 
will be removed.  
The comment was added to R5 when there were discussions of replacing MedicationStatement with MedicationUse, but MedicationUse didn't get approved.
